README.md

```diff
@@ -150,7 +151,16 @@ svc/zookeeper-client     ClusterIP   10.31.243.173   <none>        2181/TCP
 svc/zookeeper-headless   ClusterIP   None            <none>        2888/TCP,3888/TCP   2m
 ```
 
->Note: If you want to configure non deafult service accounts to zookeeper pods, refer to [this](doc/rbac.md).
+>Note: If you want to configure non deafult service accounts to zookeeper pods, set the service account inside pod
```

Comment:
could u also mention from operator version 0.2.9 onwards

Reply:
Changed that